bangazon-clumsy-penguins/BangazonAPI
# Ajw orders controller

# Name - Orders Controller

## Description
Orders controller allows front end developer access to the Orders table in BangazonAPI DB

## Steps to Test
`git checkout master`
`git fetch`
`git checkout ajw-ordersController`
`dotnet run`
in postman
`http://localhost:5000/orders` Get all Orders
`http://localhost:5000/orderspaymentTypes/{id}` Get one order
`http://localhost:5000/orderspaymentTypes/{id}?_include=products` Get one order and products on the order
`http://localhost:5000/orderspaymentTypes/{id}?_include=customers` Get one order and customer on the order
`http://localhost:5000/orders` select POST in post mane. insert `{ "customerId":  5, "customerAccountId": 6}` into the body.
`http://localhost:5000/orders/{id}` PUT paymentTypes insert `{ "customerId":  5, "customerAccountId": 7}` into the body.
`http://localhost:5000/orders/{id}` DELETE in post man, should delete the order and the products on that order
##### System configuration.
PC
##### 3rd party libraries that need to be installed.
dapper
##### Command line utilities to run.
see above
##### Link to Feature Ticket
#4
